client/mysqltest.cc

```diff
@@ -6170,6 +6171,33 @@ enum block_op find_operand(const char *start)
  return ILLEG_OP;
 }
 
+/*
+  do_break
+
+  DESCRIPTION
+  Instruction to stop execution of the current loop
+*/
+void do_break(struct st_command* command)
+{
+  int depth = 0;
+  cur_block->ok = false;
```

Comment:
Please, use the style like above at line 6155 - delete space after first part of assignment expression
int depth= 0;
cur_block->ok= false;
......and
6189 cur_block->ok= false;
6199 the same after cur_block+= depth;